app/views/course/leaderboards/groups.html.slim

```diff
@@ -0,0 +1,13 @@
+= page_header format_inline_text(@leaderboard_settings.group_leaderboard_title || t('.header'))
+
+= render partial: 'tabs'
+
+- course_groups_points = @course_groups.ordered_by_experience_points.take(display_user_count)
+= render layout: 'table', locals: { header: t('.experience_points_header'),
+                                     table_class: 'leaderboard-points' } do
+  = render partial: 'groups', locals: { course_groups: course_groups_points }
```

Comment:
A more conventional way is ( and to me it's more cleaner ):
```ruby
render partial: 'group', collection: course_groups_points
```

if you need the index, there is a variable named `group_counter` in the partial provided by rails.